metrics/test/features/dashboard/domain/entities/metrics/builds_on_date_test.dart

```diff
@@ -5,8 +5,9 @@ void main() {
   test(
     "Can't create the BuildsOnDate with the date that contains non-zero time",
     () {
+      final timestamp = DateTime(2020, 1, 1, 1, 1);
```

Comment:
what if we try to test a list of prepared data to make sure that each time component tested separately, basically we need to test cases:  fails: DateTime(2020, 1, 0, 0, 0), DateTime(2020, 0, 1, 0, 0), DateTime(2020, 0, 0, 1, 0), DateTime(2020, 0, 0, 0, 1),DateTime(2020, 1, 1, 1, 1), successful case DateTime(2020, 0, 0, 0, 0)

or maybe it would be simpler to change implementation to take date() from every DateTime passed?